src/lamp_py/migrations/versions/metadata_dev/002_26db393ea854_update_glides_location_column_names.py

```diff
@@ -43,21 +45,36 @@ def update_glides_archive(temp_dir: str, base_filename: str) -> None:
         if not file_exists:
             return
 
-        old = pq.read_table(old_local_path)
-        old_names = old.column_names
-
-        # rename columns containing gtfsID and todsID to use Id instead
-        new_names = [
-            n.replace("gtfsID", "gtfsId").replace("todsID", "todsId")
-            for n in old_names
-        ]
-        renamed = old.rename_columns(new_names)
```

Comment:
in addition to renaming, i'll also need a schema to cast against or i'll be left with whatever pyarrow converts to.

generate a list of fields and generate a new schema from that.